doc/arc/adr_006.md

```diff
@@ -1,21 +1,23 @@
-# TITLE (short noun phrase)
+# [ADR_006](../../README.md) Sharding & Routing per Geographic Region
 
-Date: DATE
+Date: 11/1
 
 ## Status
 
-proposed / accepted / superseded
+Accepted
 
 ## Context
 
-The issue motivating this decision, and any context that influences or constrains the decision.
+When trying to build a robust and highly available system, it is important to consider how to manage users from different regions. Sharding users to different deployments of a service based on their geographic location gives some natural load balancing, and ensures that users are not heavily impacted by latency to distant servers.
 
 ## Decision
 
-The change that we're proposing or have agreed to implement.
+We have chosen to have a singular deployment of the pharmacy food system.
+
+Pharmacyfood's goals at this time are to operate and expland within a single city in the near term. This does not justify the complication of managing multiple different deployments in one or more datacenters,  or dealing with sharding users across infrastructure.
 
 ## Consequences
 
-What becomes easier or more difficult to do and any risks introduced by the change that will need to be mitigated.
+Should Pharmacyfood expand to multiple citys across the united states, the nature of viewing/ordering meals and recieving recomendations will likely not be heavily impacted by higher latency operations due to geographic distance.
 
-(For more information, see: https://cognitect.com/blog/2011/11/15/documenting-architecture-decisions)
+Business wise, it would be a very great problem for Pharmacyfoods to have if a singular deployment is not scaling well enough. That bridge can be crossed later.
```

Comment:
```suggestion
Business wise, it would be a very great problem for farmacyfoods to have if a singular deployment is not scaling well enough. That bridge can be crossed later.
```